dizoo/dmc2gym/entry/dmc2gym_onppo_main.py

```diff
@@ -0,0 +1,122 @@
+import os
+from easydict import EasyDict
+from functools import partial
+from tensorboardX import SummaryWriter
+import dmc2gym
+
+from ding.worker import BaseLearner, SampleSerialCollector, InteractionSerialEvaluator
+from ding.model import VAC
+from ding.policy import PPOPolicy
+from ding.envs import DingEnvWrapper, FinalEvalRewardEnv, BaseEnvManager
+from ding.config import compile_config
+from ding.utils import set_pkg_seed
+from dizoo.dmc2gym.config.dmc2gym_ppo_config import cartpole_balance_ppo_config
+from dizoo.dmc2gym.envs.dmc2gym_env import *
+
+
+class InfoWrapper(gym.Wrapper):
+    def __init__(self, env, cfg):
+        super().__init__(env)
+        cfg = EasyDict(cfg)
+        self._cfg = cfg
+        self._observation_space = dmc2gym_env_info[cfg.domain_name][cfg.task_name]["observation_space"](
+            from_pixels=self._cfg["from_pixels"],
+            height=self._cfg["height"],
+            width=self._cfg["width"],
+            channels_first=self._cfg["channels_first"]
+        )
+        self._action_space = dmc2gym_env_info[cfg.domain_name][cfg.task_name]["action_space"]
+        self._reward_space = dmc2gym_env_info[cfg.domain_name][cfg.task_name]["reward_space"](self._cfg["frame_skip"])
+
+    def seed(self, seed: int, dynamic_seed: bool = True) -> None:
+        self._seed = seed
+        self._dynamic_seed = dynamic_seed
+        np.random.seed(self._seed)
+
+    def _process_obs(self, obs):
+        if self._cfg["from_pixels"]:
+            obs = to_ndarray(obs).astype(np.uint8)
+        else:
+            obs = to_ndarray(obs).astype(np.float32)
+        return obs
+
+    def step(self, action):
+        action = np.array([action]).astype('float32')
+        obs, reward, done, info = self.env.step(action)
+        return self._process_obs(obs), reward, done, info
+
+    def reset(self):
+        obs = self.env.reset()
+        return self._process_obs(obs)
+
+
+def wrapped_dmc2gym_env(cfg):
+    default_cfg = {
+            "frame_skip": 3,
+            "from_pixels": True,
+            "visualize_reward": False,
+            "height": 100,
+            "width": 100,
+            "channels_first": True,
+    }
+    default_cfg.update(cfg)
+
+
+    return DingEnvWrapper(
+        dmc2gym.make(
+            domain_name=default_cfg["domain_name"],
+            task_name=default_cfg["task_name"],
+            seed=1,
+            visualize_reward=default_cfg["visualize_reward"],
+            from_pixels=default_cfg["from_pixels"],
+            height=default_cfg["height"],
+            width=default_cfg["width"],
+            frame_skip=default_cfg["frame_skip"]
+        )
+        ,
+        cfg={
+            'env_wrapper': [
+                lambda env: InfoWrapper(env, default_cfg),
+                lambda env: FinalEvalRewardEnv(env),
+            ]
+        }
+    )
+
+
+def main(cfg, seed=0, max_iterations=int(1e10)):
```

Comment:
don't use `max_iterations` argument but set `max_env_step` like that in `serial_pipeline`